locales/en/plugin__nmstate-console-plugin.json

```diff
@@ -49,9 +49,10 @@
   "Interface name": "Interface name",
   "Interface name should follow the linux kernel naming convention. The name should be smaller than 16 characters.": "Interface name should follow the linux kernel naming convention. The name should be smaller than 16 characters.",
   "Interface name should follow the linux kernel naming convention. Whitespaces and slashes are not allowed.": "Interface name should follow the linux kernel naming convention. Whitespaces and slashes are not allowed.",
+  "IP address": "IP address",
   "IP configuration": "IP configuration",
   "IPv4": "IPv4",
-  "IPV4": "IPV4",
+  "IPV4 address": "IPV4 address",
```

Comment:
Is there a reason why the 'v' is capitalized here or is it just a typo?

Reply:
no, i think its a typo. I would fix it on the upstream first